src/elm/Page/Community/Settings/Settings.elm

```diff
@@ -0,0 +1,149 @@
+module Page.Community.Settings.Settings exposing (Model, Msg, init, msgToString, update, view)
+
+import Api.Graphql
+import Community
+import Eos exposing (Symbol)
+import Graphql.Http
+import Html exposing (Html, a, button, div, span, text)
+import Html.Attributes exposing (class, style)
+import I18Next exposing (t)
+import Page exposing (viewHeader)
+import Route exposing (Route)
+import Session.LoggedIn as LoggedIn exposing (External(..))
+import Session.Shared exposing (Shared)
+import UpdateResult as UR
+
+
+init : LoggedIn.Model -> Symbol -> ( Model, Cmd Msg )
+init { shared } symbol =
+    ( initModel symbol
+    , Api.Graphql.query shared (Community.communityQuery symbol) CompletedLoad
```

Comment:
I was able to access this page even without being the community creator. Please take notice that this was specified in the issue description:

> Its important to make this screen accessible only by community admins. You can check the community creator, that is the admin account. Make sure to hide the link from the community page as well as to block users that directly use the URL